AGGREGATE.md

```diff
@@ -72,22 +72,18 @@ example an ad-tech will use the API to collect:
 Registering sources eligible for aggregate reporting entails adding a new header
 on the response to the `attributionsrc` request:
 `Attribution-Reporting-Register-Aggregatable-Source`, in the form of a JSON
-list.
+dict.
 ```jsonc
-[{
-  // Generates a "0x159" key piece (low order bits of the key) for the key named
-  // "campaignCounts"
-  "id": "campaignCounts",
-  "key_piece": "0x159" // User saw ad from campaign 345 (out of 511)
-},
 {
-  // Generates a "0x5" key piece (low order bits of the key) for the key named "geoValue"
-  "id": "geoValue",
-  // Source-side geo region = 5 (US), out of a possible ~100 regions.
-  "key_piece": "0x5"
-}]
+  // Generates a "0x159" key piece (low order bits of the key) for the key named
+  // "campaignCounts".
+  "campaignCounts": "0x159", // User saw ad from campaign 345 (out of 511)
```

Comment:
Did we decide whether to use a string here or a dictionary to support future changes?

CC @johnivdel @csharrison

Reply:
Updated the issue, let's go with a string to keep that schema as simple as possible.

Reply:
SGTM, thanks!